package.json

```diff
@@ -16,38 +17,39 @@
     "autoprefixer": "10.4.15",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
-    "geist": "^1.0.0",
+    "geist": "^1.2.2",
```

Comment:
Is there a reason why some of these packages were upgraded? We need to be very careful when upgrading versions.

Reply:
I don't recall upgrading anything when I was making changes, I did have to pull from develop's new changes in regards to the Docker testing (removing Home component and test folder)

Reply:
I believe this happened because there wasn't a package.lock file that was committed. So your installation command installed and updated to the latest packages. Once a package.lock file is in our repo, the only time we should upgrade our packages are when we need them to be upgraded. And not an automatic script that does it for us.